### DISCUSSION TITLE
Tamagui

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#193] [Announcements]
- https://twitter.com/tamagui_js
- https://twitter.com/natebirdman
- https://tamagui.dev/

## Intro

`Andrew`: Hello and welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

`Justin`: Hey folks, our guest to today is Nate Wienert. Nate is the creator of [tamagui](https://tamagui.dev/), a UI kit for building cross web/react native apps. Nate, it's great to have you! Before we dig in, would you like to tell our listeners a bit more about yourself?

### Tamagui

- [x] what is tamagui 
- [x] where did you get inspired for your component api? How do they work under the hood
- [x] in practice how do you actually support both native and web
- [x] what is the optimizing compiler and what does it do? 
- [ ] What are some cool features the compiler enables?
- [x] is it accessible cross platform? Was that support hard to implement?
- [x] how much does it adapt to each platform if I style my elements?
- [x] how does it help you write cross platform animations?
- [x] the docs mention server components. What are you doing with that in tamagui?
- [x] how is it so fast compared to the alternatives?
- [x] can I share my navigation across web and native? Do I have to create unique apps for each?
- [x] what are some of the benefits of building an app like this? When should you not?

### Future

- [x] what is your vision for tamagui studio?
- [x] what is takeout?

**Comment by hipstersmoothie** [comment]
https://vaul.emilkowal.ski/
https://github.com/supabase/postgres_lsp

**Comment by just-be-dev** [comment]
- https://twitter.com/ertekinno/status/1686336911869173760
- https://docs.dxos.org/

**Comment by natew** [comment]
https://gqty.dev
https://uncut.wtf
